languages/python/exercises/concept/bools/arcade_game_test.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+from arcade_game import eat_ghost, score, lose, win
+
+
+class TestArcadeGame(unittest.TestCase):
+
+    def test_ghost_gets_eaten(self):
+        self.assertEqual(
```

Comment:
When comparing to `True`, `False`, or `None`, the preferred function is `TestCase.assertIs`:
```suggestion
        self.assertIs(
```